src/utils/Draft.js

```diff
@@ -1,14 +1,32 @@
 import { csrfFetch } from 'utils/CSRF';
-import { arrayIsSubset, arrayShuffle } from 'utils/Util';
-import { COLOR_COMBINATIONS, inclusiveCount } from 'utils/Card';
+import { arrayIsSubset, arrayShuffle, fromEntries } from 'utils/Util';
+import { COLOR_COMBINATIONS } from 'utils/Card';
 
 let draft = null;
 
+const toValue = (elo) => 10 ** (elo / 400);
+
+function addSeen(seen, cards) {
+  for (const card of cards) {
+    const colors = card.colors ?? card.details.colors ?? [];
+    if (colors.length > 0) {
```

Comment:
No card rating needs to be added for colorless cards or do they have a color?  'C'?

Reply:
Colorless gets added to every combination so just reduces variance so they get skipped here.